CompilerSource/compiler/jdi_utility.cpp

```diff
@@ -57,8 +57,10 @@ static void iterate_overloads(
   }
 
   for (definition_template* templateOverload : d->template_overloads) {
-    if (min > templateOverload->params.size()) min = templateOverload->params.size();
-    if (max < templateOverload->params.size()) max = templateOverload->params.size();
+    definition* def = templateOverload->def;
+    if (def->flags & DEF_OVERLOAD) {
+     visit_overload(static_cast<definition_overload*>(def), min, max, varargs_t);
+    }
   }
```

Comment:
@JoshDreamland does this look more correct?

Reply:
Yes, although I can't remember if it's going to contain an overload or a complete function. I recall spending effort on that, so maybe I did get it to be an overload, but it wouldn't surprise me if it were still a complete function that just happens to have one overload in it.

But anyway, if that code solves the problem, great, because it looks correct.